apidemo/views.py

```diff
@@ -8,15 +9,52 @@
 from .categoryModel import Category
 from .userModel import User
 from .articlesModel import Articles
-from rest_framework.permissions import BasePermission ,AllowAny
+import time
+import jwt
+from datetime import datetime,timedelta,timezone
+key="tuan"
 from rest_framework.decorators import APIView
 
 # ------------------------Start User -----------------
 
-class UserApiGetAll(APIView):
-    permission_classes = [AllowAny]
-    def get(self,request):
+def CheckAuth(a,CheckArticles=0):
+    try:
         
+        jwt_payloadSplit=a['Authorization'].split(' ')
+        jwt_payload = jwt_payloadSplit[1]
+        jwt_payloadDecode=jwt.decode(jwt_payload, key, algorithms=["HS256"])
+        if jwt_payloadDecode['Group']=="admin":
+            return 1
+        else:
+            if(CheckArticles):
+                print('vcl')
+                return 1
+            else:
+                    return 0
+    except:
+        return 0
+
+class TokenView(APIView):
```

Comment:
Each view should be separated into its own class. I.e: tokenView.py, userView.py, ...